content/reference/drops/contact_request_drop.md

```diff
@@ -0,0 +1,25 @@
+# Contact Request Drop
+
+Allows to check and show Contact Form errors.
```

Comment:
@hash4di Most of the drops below miss this convention.
Decide to use it everywhere or nowhere.